### PR TITLE
Hook to get img url and Addhouse fix

### PR DESCRIPTION
**Pull request by FebernalGober**
The SaveImgUrl Hook was created to upload the house images to Firebase and the GetCurrentUser Hook was updated to show the UID instead of the email

# Description

The SaveImgUrl Hook was created to upload the house images to Firebase and the GetCurrentUser Hook was updated to show the UID instead of the email

### CODE REVIEW

File: renta_facil_colima/package-lock.json
Comment (ecruz17) [inline renta_facil_colima/package-lock.json:1]: I'd suggest to include this into the .gitignore

File: renta_facil_colima/yarn.lock
Comment (ecruz17) [inline renta_facil_colima/yarn.lock:1]: be careful with using both npm and yarn! include this into the .gitignore

**Review by ecruz17 — APPROVED** [review]
LGTM wuacho

**Review by BrandonVG — APPROVED** [review]

File: renta_facil_colima/src/hooks/AddHouse.js [hunk 1]
```diff
@@ -21,6 +18,8 @@ try {
     bathrooms: bathrooms, 
     surface: surface,
     rent: rent,
+    //img: img, para el futuro//
```
Comment (YilKanda) [inline renta_facil_colima/src/hooks/AddHouse.js:23]: Do not leave code in comments.

File: renta_facil_colima/src/hooks/AddHouse.js [hunk 2]
```diff
@@ -21,6 +18,8 @@ try {
...
+    userId: GetCurrentUser()
   });
 
   console.log("Document written with ID: ", docRef.id);
```
Comment (YilKanda) [inline renta_facil_colima/src/hooks/AddHouse.js]: Do not leave debug code on GitHub.

File: renta_facil_colima/src/hooks/GetCurrentUser.js [hunk 3]
```diff
@@ -4,11 +4,11 @@ const GetCurrentUser = () => {
   const user = auth.currentUser;
 
   if (user) {
-    console.log(user.email);
-    return user.email;
+    console.log(user.uid);
```
Comment (YilKanda) [inline renta_facil_colima/src/hooks/GetCurrentUser.js]: Do not leave debug code on GitHub.

File: renta_facil_colima/src/hooks/GetCurrentUser.js [hunk 4]
```diff
@@ -4,11 +4,11 @@ const GetCurrentUser = () => {
...
+    return user.uid;
   } else {
     console.log("No user is signed in.");
```
Comment (YilKanda) [inline renta_facil_colima/src/hooks/GetCurrentUser.js]: Do not leave debug code on GitHub.

File: renta_facil_colima/src/hooks/SaveImgUrl.js [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+            const url = await imageRef.getDownloadURL();
+            setDownloadUrl(url);
+        },
+        );
+    } catch (error) {
+        setError(error);
+    }
+    };
+
+    return {uploadImage, uploadProgress, downloadUrl, error};
```
Comment (YilKanda) [inline renta_facil_colima/src/hooks/SaveImgUrl.js]: Check the indentation of your code.

File: renta_facil_colima/src/hooks/AddHouse.js (hunk 2 above)
Comment (FebernalGober) [reply renta_facil_colima/src/hooks/AddHouse.js]: Done

File: renta_facil_colima/src/hooks/GetCurrentUser.js (hunk 3 above)
Comment (FebernalGober) [reply renta_facil_colima/src/hooks/GetCurrentUser.js]: Done

File: renta_facil_colima/src/hooks/GetCurrentUser.js (hunk 4 above)
Comment (FebernalGober) [reply renta_facil_colima/src/hooks/GetCurrentUser.js]: Done

File: renta_facil_colima/src/hooks/SaveImgUrl.js (hunk 5 above)
Comment (FebernalGober) [reply renta_facil_colima/src/hooks/SaveImgUrl.js]: done

**Review by DerekAyala — APPROVED** [review]

**Review by FebernalGober — COMMENTED** [review]
Done

**Pull request merged**